src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -228,6 +264,12 @@ ErrorMessageOr<std::unique_ptr<PdbFile>> PdbFileLlvm::CreatePdbFile(
     return ErrorMessage(absl::StrFormat("Unable to load PDB file %s with error: %s",
                                         file_path.string(), llvm::toString(std::move(error))));
   }
+
+  if (!PdbHasDbiStream(session.get())) {
```

Comment:
What do you think of still adding a comment regarding the fact that the Dbi Stream is for `GetAge`?

Reply:
Done.